src/formElementPlugins/fileUpload/constants.ts

```diff
@@ -1,9 +1,9 @@
 export default {
-  ERROR_FILE_ALREADY_UPLOADED: 'File already uploaded',
-  ERROR_FILE_TYPE_NOT_PERMITTED: 'File type not permitted',
-  ERROR_FILE_TOO_BIG: 'File too big',
-  ERROR_TOO_MANY_FILES: 'Too many files!',
-  ERROR_UPLOAD_PROBLEM: 'Problem uploading file',
-  BUTTON_CLICK_TO_UPLOAD: 'Click to upload',
-  BUTTON_UPLOAD_ANOTHER: 'Upload another',
+  ERROR_FILE_ALREADY_UPLOADED: 'O mesmo arquivo já foi carregado',
+  ERROR_FILE_TYPE_NOT_PERMITTED: 'O tipo do arquivo não é permitido',
+  ERROR_FILE_TOO_BIG: 'O tamanho do arquivo não é permitido',
+  ERROR_TOO_MANY_FILES: 'Não é permitido carregar mais arquivos',
+  ERROR_UPLOAD_PROBLEM: 'Problema ao carregar arquivo',
+  BUTTON_CLICK_TO_UPLOAD: 'Carregar arquivo',
+  BUTTON_UPLOAD_ANOTHER: 'Carregar outro arquivo',
```

Comment:
Doesn't this need an "_en"/ "_pt" version rather than over-writing the English?

Reply:
Hi Carl, yes. We could do something more cleaver about these strings later. For now I was just adding then so I can have this on a demo instance that I'm going to build from this PR.

We shouldn't be merging with over-writing.